src/components/renderer/Tooltip.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import { Tooltip as MuiTooltip } from '@mui/material'
+import useOneBlinkFormContainer from '../../hooks/useOneBlinkFormContainer'
+
+function Tooltip({
+  children,
+  ...props
+}: React.PropsWithChildren<React.ComponentProps<typeof MuiTooltip>>) {
```

Comment:
```suggestion
}: React.ComponentProps<typeof MuiTooltip>) {
```

`<MuiTooltip />` will already has a `children` prop defined